src/makeCalculator.js

```diff
@@ -38,6 +38,39 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      return calculator.result + value;
+    },
```

Comment:
This is method that should add two values and return result of calculation
It shouldn't have access to parent object and it's properties